### DISCUSSION TITLE
A New Space

### DISCUSSION THREAD

**Discussion by Hap-404** [community/community#47455] [GitHub Education]
Hello Everyone, I just started using GitHub, so I am new here and would love to learn new things and share my knowledge with you guys, Help me with this amazing thing called Github and Let's collaborate on different things.

**Comment by Ntshangase** [comment]
Hi, I'm always ready to assist.

**Comment by Lamparter** [comment]
> Hello Everyone, I just started using GitHub, so I am new here and would love to learn new things and share my knowledge with you guys, Help me with this amazing thing called Github and Let's collaborate on different things.

Hello @Hap-404.
Good evening/morning/afternoon/night and welcome to the GitHub community! 🎉

---

Here are some starting tips to get you collaborating 😄
- [`take a look at the explore page`](https://github.com/explore) (and take a look at some interesting projects to have a look at!)
- [`browse trending projects`](https://github.com/trending)
- know what code you're interested in? [`take a look at different projects by topic`](https://github.com/topics)
- [`personalise your profile: display name and set an avatar`](https://github.com/settings/profile)
- [`familiarise yourself with the way github works`](https://github.com/features)
- **looks like you're a student, too**, I see you've activated your free GitHub pro, but you can also [`checkout more of your benefits`](https://education.github.com/benefits) if you like.
- **get coding**!! [`create a new repository (github speak for code library) 🎉`](https://github.com/new)
